src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderAssessmentResourceIntTest.kt

```diff
@@ -363,6 +448,149 @@ class OffenderAssessmentResourceIntTest : ResourceTest() {
   @Nested
   @DisplayName("POST /api/offender-assessments/category/categorise")
   inner class CreateCategorisation {
+
+    @Nested
+    inner class Authorisation {
+
+      @Test
+      fun `should return 401 when user does not even have token`() {
+        webTestClient.post().uri("/api/offender-assessments/category/categorise")
+          .header("Content-Type", APPLICATION_JSON_VALUE)
+          .bodyValue(
+            """
+            {
+              "bookingId" : -35,
+              "category": "D",
+              "committee": "RECP"
+            }
+            """,
+          )
+          .exchange()
+          .expectStatus().isUnauthorized
+      }
+
+      @Test
+      fun `should return 403 if does not have authorised role`() {
+        webTestClient.post().uri("/api/offender-assessments/category/categorise")
```

Comment:
At line 168 there's an endpoint that returns a 404 if no authorised role. Just wondering what the difference is with this endpoint whcih returns 403?

Reply:
Good question - there is another PR coming thru - this is because it is using VerifyAgencyAccess which hasn't yet been promoted/updated to change the EntityNotFound to an AccessDenied (only VerifyBookingAccess/VerifyOffenderAccess have been updated so far).  This test will enable me to check for a 403 once the update has been made.